diff --git a/src/arithmetic/big_native/primes.rs b/src/arithmetic/big_native/primes.rs
index 314af2d..5b9882b 100644
--- a/src/arithmetic/big_native/primes.rs
+++ b/src/arithmetic/big_native/primes.rs
@@ -503,7 +503,7 @@ fn is_bit_set(x: &BigUint, i: usize) -> bool {
 fn gen_biguint_below<R: Rng>(r: &mut R, upper: &BigUint) -> BigUint {
     loop {
         let bits = upper.bits();
-        let bytes = bits.div_ceil(&8);
+        let bytes = bits.div_ceil(8);
         let mut buf = vec![0u8; bytes as usize];
         r.fill_bytes(&mut buf);
 
diff --git a/src/lib.rs b/src/lib.rs
index 9c5df94..9194d72 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
 /*
     This file is part of Curv library
     Copyright 2018 by Kzen Networks
